holder.yml

```
@@ -22,7 +22,9 @@ paths:
        - oAuth2: []
        - zCap: []
       summary: Gets a credential or verifiable credential by ID. To get a credential that does not have credential.id set but has an associated credentialId value, pass credentialId instead.
-      x-expectedCaller: "Issuer Coordinator"
+      x-expectedCaller:
+        - Holder Coordinator
+        - Issuer Coordinator
```

A workflow service could also conceivably use this endpoint.

```suggestion
        - Issuer Coordinator
        - Workflow Service
```